.circleci/config.yml

```diff
@@ -52,7 +52,7 @@ jobs:
           name: Pull Deps
           command: |
             source ${INIT_ENV}
-            ./src/CARMAPlatform/docker/checkout.sh ${PWD}
+            ./src/CARMAPlatform/docker/checkout.bash -r ${PWD}
```

Comment:
I believe this directory should be carma-platform.

Reply:
So when we were doing the name change we discussed this and decided there wasn't a need to change the directory names used in CI, but we can change it now if we'd like. The dir is right though and is created two steps prior in config.yml

Reply:
No need.  Just wanted to make sure it's not broken.